entries/p/payback.de.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Payback": {
+    "domain": "payback.de",
+    "url": "https://www.payback.de/",
+    "tfa": [
+      "totp",
+      "custom-software"
```

Comment:
Is it true TOTP RFC-6238 or does it only work via the PAYBACK app?

Reply:
Nope, just custom software, no "real TOTP"